Products/ZenModel/migrate/migrateLocalhostPerf.py

```
@@ -0,0 +1,60 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2011, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+
+__doc__='''
+
+Convert 4.x-style DistributedPerformanceConf localhost hub + collector to
+a 5.x-style one.
+
+'''
+import Migrate
+import logging
+
+log = logging.getLogger('zen.migrate')
+
+class MigrateLocalhostPerf(Migrate.Step):
+    version = Migrate.Version(5, 0, 70)
+
+    def cutover(self, dmd):
+        try:
+            from ZenPacks.zenoss.DistributedCollector import \
+                addHubRoot, addLocalhostHub, convertLocalhostHubToCC, convertAllMonitors, \
+                convertMonitor, convertLocalhostMonitorToCC
+        except ImportError:
+            log.info("Required distributed collector module not found, skipping")
```

I wouldn't say "Required" here, just silently skip the migrate